semantickernel-experimental/src/main/java/com/microsoft/semantickernel/data/vectorstorage/attributes/VectorStoreRecordVectorAttribute.java

```diff
@@ -31,5 +32,8 @@
     /**
      * Distance function to be used for to compute the distance between vectors.
      */
-    String distanceFunction() default "";
+    DistanceFunction distanceFunction() default DistanceFunction.COSINE_DISTANCE;
```

Comment:
This should be `null` as not all vector collections support `COSINE_DISTANCE`. The default for Azure AI Search is `COSINE_SIMILARITY` for example. Each store should decide its default value.